website/app/components/doc/page/cover.js

```diff
@@ -1,10 +1,30 @@
 import Component from '@glimmer/component';
 
 const STATUSES = {
+  'on-roadmap': {
```

Comment:
[non-blocking question] where did this list come from?

Reply:
> [non-blocking question] where did this list come from?

https://www.figma.com/file/Ky0qWjvHZR3je1lCBlzNB1/HDS-website?node-id=1345%3A79255&t=49uswjcCEF4o26sh-1

<img width="390" alt="screenshot_2224" src="https://user-images.githubusercontent.com/686239/208447612-55acf9fc-546e-419e-b791-11ca1a811b9b.png">

Reply:
@heatherlarsen it looks like this is introducing something that was previously design-only to the broader documentation effort. I'd like to review the intentions for this before launch to ensure it makes sense to include this in the release so that ourselves and consumers are on the same page about how we use these.

Reply:
@Dhaulagiri since it's not blocking and can be easily changed, I am merging the PR for now, and I'll let you and @heatherlarsen figure out the final list of possible "status" variants for the page cover.